rules/windows/process_creation/proc_creation_win_apt_babyshark.yml

```diff
@@ -21,9 +21,9 @@ logsource:
 detection:
     selection:
         CommandLine:
-            - reg query "HKEY_CURRENT_USER\Software\Microsoft\Terminal Server Client\Default"
-            - powershell.exe mshta.exe http*
-            - cmd.exe /c taskkill /im cmd.exe
+            - 'reg query "HKEY_CURRENT_USER\Software\Microsoft\Terminal Server Client\Default"'
+            - 'powershell.exe mshta.exe http*'
```

Comment:
'...exe http*' ? not contains or startswith

Reply:
I changed it to contains